ts/features/euCovidCert/screens/EuCovidCertRevokedScreen.tsx

```diff
@@ -1,17 +1,54 @@
 import * as React from "react";
 import { connect } from "react-redux";
 import { Dispatch } from "redux";
-import { H1 } from "../../../components/core/typography/H1";
+import { Image } from "react-native";
+import { View } from "native-base";
 import { GlobalState } from "../../../store/reducers/types";
+import { InfoScreenComponent } from "../../../components/infoScreen/InfoScreenComponent";
+import I18n from "../../../i18n";
+import revokedImage from "../../../../img/features/euCovidCert/certificate_revoked.png";
+import Markdown from "../../../components/ui/Markdown";
+import { Link } from "../../../components/core/typography/Link";
+import { openWebUrl } from "../../../utils/url";
+import { euCovidCertificateUrl } from "../../../urls";
 import { BaseEuCovidCertificateLayout } from "./BaseEuCovidCertificateLayout";
 
 type Props = ReturnType<typeof mapDispatchToProps> &
-  ReturnType<typeof mapStateToProps>;
+  ReturnType<typeof mapStateToProps> & { revokeInfo?: string };
 
-const EuCovidCertRevokedScreen = (_: Props): React.ReactElement => (
+const EuCovidCertRevokedContentComponent = (props: Props) => (
+  <>
+    <View spacer extralarge />
+    <View spacer extralarge />
+    <InfoScreenComponent
+      image={
+        <Image
+          source={revokedImage}
+          importantForAccessibility={"no"}
+          accessibilityElementsHidden={true}
+          style={{ width: 104, height: 104, resizeMode: "contain" }}
+        />
+      }
+      title={I18n.t("features.euCovidCertificate.revoked.title")}
+      body={
+        <Link
+          accessibilityRole={"link"}
+          accessibilityHint={I18n.t("global.accessibility.linkHint")}
+          onPress={() => openWebUrl(euCovidCertificateUrl)}
+        >
+          {I18n.t("features.euCovidCertificate.common.learnMore")}
+        </Link>
+      }
+    />
+    <View spacer />
+    {props.revokeInfo && <Markdown>{props.revokeInfo}</Markdown>}
```

Comment:
Perhaps we should handle the messageID copy from markdown
https://pagopa.invisionapp.com/console/App_IO---eucovidcert-ckp8db5h70pq3016bgnkkddka/ckpcl1yjt01fe01xa29rl4qz4/play#project_console

```suggestion
    {props.revokeInfo && <Markdown
        onLinkClicked={(link: string) => {
          deriveCustomHandledLink(link).map(hl => {
            if (hl.schema === "copy") {
              clipboardSetStringWithFeedback(hl.value);
              return;
            }
            void taskLinking(hl.url).run();
          });
        }}
      >
        {props.revokeInfo}
      </Markdown>}
```

Reply:
I didn't figured out to handle it by component size since I imagined it could be handled by a custom url like "ioit://COPY..." or something similar. My bad, I'm going to fix in next commit 😉

Reply:
@undermaken @fabriziofff 7f21ce6 solves

I've added an handler in case of error while opening the URL